modules/persistent-storage-csi-migration-sc.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-migration.adoc
+
+:_content-type: CONCEPT
+[id="persistent-storage-csi-migration-sc-implications_{context}"]
+= Storage class implications
+
+For new {product-title} 4.13, and later, installations, the default storage class is the CSI storage class. All volumes provisioned using this storage class are CSI persistent volumes (PVs).
+
+For clusters upgraded from 4.12, and earlier, to 4.13, and later, the CSI storage class is created, and is set as the default if no default storage class was set prior to the upgrade. In the very unlikely case that there is a storage class with the same name, the existing storage class remains unchanged. Any existing in-tree storage classes remain, and might be necessary for certain features, such as volume expansion to work for existing in-tree PVs. While storage class referencing to the in-tree storage plugin will continue working, we recommend that you switch the default storage class to the CSI storage class.
```

Comment:
to 4.13 and later. I would remove "and later", we plan to have it enabled for all clusters in 4.14.

Reply:
Should we document a procedure to `switch the default storage class to the CSI storage class`?

Reply:
@sferich888 See Gregory's comment above about adding a link.

Reply:
If we can link to this that would be great; but I know our styling  and tooling may not allow for that.

Reply:
Link is in.